plugins/source/aws/policies/queries/cloudwatch/alarm_organization_changes.sql

```diff
@@ -0,0 +1,31 @@
+insert into aws_policy_results
+select
+  :'execution_time' as execution_time,
+  :'framework' as framework,
+  :'check_id' as check_id,
+  'Ensure a log metric filter and alarm exists for AWS Organizations changes (Automated)' as title,
+  account_id,
+  cloud_watch_logs_log_group_arn as resource_id,
+  case when
+      pattern = '{ ($.eventName = AcceptHandshake) '
+      || '|| ($.eventName = AttachPolicy) '
+      || '|| ($.eventName = CreateAccount) '
+      || '|| ($.eventName = CreateOrganizationalUnit) '
+      || '|| ($.eventName = CreatePolicy) '
+      || '|| ($.eventName = DeclineHandshake) '
+      || '|| ($.eventName = DeleteOrganization) '
+      || '|| ($.eventName = DeleteOrganizationalUnit) '
+      || '|| ($.eventName = DeletePolicy) '
+      || '|| ($.eventName = DetachPolicy) '
+      || '|| ($.eventName = DisablePolicyType) '
+      || '|| ($.eventName = EnablePolicyType) '
+      || '|| ($.eventName = InviteAccountToOrganization) '
+      || '|| ($.eventName = LeaveOrganization) '
+      || '|| ($.eventName = MoveAccount) '
+      || '|| ($.eventName = RemoveAccountFromOrganization) '
+      || '|| ($.eventName = UpdatePolicy) '
+      || '|| ($.eventName = UpdateOrganizationalUnit) }'
+      then 'pass'
+      else 'fail'
+  end as status
+from view_aws_log_metric_filter_and_alarm
```

Comment:
I don't think that the "unit" of compliance is individual resources, but the account as a whole. So rather than marking all log_metrics as failing we should only be marking the account as pass/fail